.pre-commit-config.yaml

```diff
@@ -7,3 +7,11 @@ repos:
     rev: 6.1.0
     hooks:
     -   id: flake8
+-   repo: local
+    hooks:
+    -   id: pyright
+        name: pyright
+        entry: poetry run pyright --warnings src
+        pass_filenames: false
```

Comment:
with `pass_filenames: true` we start one pyright process _per changed file_ in parallel which kills my cpu :(

Reply:
`poetry run pyright`
Ehhh, not nixos-compatible x)

Reply:
Nvm, is working